packages/usa-button-group/src/test/test-patterns/usa-button-group--test-nested.twig

```diff
@@ -0,0 +1,49 @@
+<div class="usa-prose measure-5 padding-2 border-1px border-gray-20 margin-bottom-4">
+  <p class="font-body-lg text-bold margin-top-0">Test nested button groups</p>
+  <p>Confirm that buttons inside nested button groups are:</p>
+  <ul>
+    <li>The same height as their sibling buttons</li>
+    <li>The same height as their parent buttons</li>
+  </ul>
+</div>
+
+<h3>Nested button group</h3>
+<ul class="usa-button-group">
+  <li class="usa-button-group__item">
+    <a href="#" class="usa-button usa-button--outline">Button</a>
+  </li>
+  <li class="usa-button-group__item">
+    <a href="#" class="usa-button usa-button--outline maxw-15">Button with long text </a>
+  </li>
+  <li class="usa-button-group__item">
+    <ul class="usa-button-group">
+      <li class="usa-button-group__item ">
+        <a href="#" class="usa-button usa-button--outline maxw-10">Nested Button</a>
+      </li>
+      <li class="usa-button-group__item">
+         <button type="button" class="usa-button">Nested Button</button>
+      </li>
+    </ul>
+  </li>
+</ul>
+
+<h3>Nested button group - Segmented</h3>
+<ul class="usa-button-group usa-button-group--segmented">
+  <li class="usa-button-group__item">
+    <a href="#" class="usa-button usa-button--outline">Button</a>
+  </li>
+  <li class="usa-button-group__item">
+    <a href="#" class="usa-button usa-button--outline maxw-15">Button with long text </a>
+  </li>
+  <li class="usa-button-group__item">
+    <ul class="usa-button-group usa-button-group--segmented">
+      <li class="usa-button-group__item">
+        <a href="#" class="usa-button usa-button--outline maxw-10">Nested Button</a>
+      </li>
+      <li class="usa-button-group__item">
+         <button type="button" class="usa-button">Nested Button</button>
```

Comment:
I noticed that this was a `button` element while the rest of the buttons are styled anchor elements.

Was there a reason for this? Was it to match the example provided in the issue?

Reply:
I switched out most of the anchors to buttons and clearly labeled each in [f4e7bd7](https://github.com/uswds/uswds/pull/5885/commits/f4e7bd7e25e0cca4d6dedfd3ced14196afcaf14b)